src/sentry/models/exporteddata.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import absolute_import
+
+from datetime import timedelta
+from django.conf import settings
+from django.db import models
+from django.utils import timezone
+from uuid import uuid4
+
+from sentry.db.models import FlexibleForeignKey, Model, sane_repr
+
+DEFAULT_EXPIRATION = timedelta(days=7)
+
+
+def generate_data_id():
+    return uuid4().hex
+
+
+class ExportedData(Model):
+    """
+    Stores references to asynchronous data export jobs being stored
+    in the Google Cloud Platform temporary storage solution.
+    """
+
+    __core__ = False
+
+    organization = FlexibleForeignKey("sentry.Organization")
+    user = FlexibleForeignKey(settings.AUTH_USER_MODEL)
+    data_id = models.CharField(max_length=32, unique=True, default=generate_data_id)
```

Comment:
What will we actually use this id for? It might make sense to just use the pk of the table. I believe Django also has a `UUIDField` that we could use instead.

Reply:
I think it makes sense to use the postgres id's you're probably right